wagtail_ab_testing/models.py

```diff
@@ -94,14 +151,14 @@ def add_participant(self, variant=None):
 
         return variant
 
-    def log_conversion(self, variant):
+    def log_conversion(self, variant, *, time=None):
```

Comment:
`*` -> `*args` just for good practice?

Reply:
I don't think that's good practice! (unless you need to use the args) I think it's better to have a tighter signature that'll give errors if too many arguments were passed.